prow/github/types.go

```diff
@@ -297,6 +298,13 @@ type BranchProtectionRequest struct {
 	Restrictions               *Restrictions               `json:"restrictions"`
 }
 
+func (r BranchProtectionRequest) String() string {
+	if bytes, err := json.Marshal(&r); err == nil {
+		return string(bytes)
```

Comment:
shouldn't we indent the err case per style guidelines?

Reply:
ACK